experiences.js

```diff
@@ -0,0 +1,98 @@
+const experiences = document.querySelector('listedexperiences');

```

Comment:
I tried to use this file and use variables and HTML injection to put this text in the webpage, but it did not work.